dictionaries/npm/cspell-ext.json

```diff
@@ -17,6 +17,24 @@
       "dictionaries": ["npm"]
     }
   ],
+  "ignoreRegExpList": ["npm-scripts"],
+  "patterns": [
+    {
+      "name": "npm-scripts",
+      "pattern": ["npm-scripts-pre", "npm-scripts-post"],
+      "description": "Matches NPM scripts pre and post from NPM lifecycle."
+    },
+    {
+      "name": "npm-scripts-pre",
+      "pattern": "/pre[^\"]+/",
+      "description": "Matches NPM scripts pre from NPM lifecycle."
+    },
+    {
+      "name": "npm-scripts-post",
+      "pattern": "/post[^\"]+/",
```

Comment:
@Jason3S Let me know if my explanation is more helpful as why I chose RegExp.

Reply:
Thank you, now I understand the intent. The current regexp is a bit too broad. It will match more than what is intended. Example: 
text:  `"description": "This package helps with post processing of dynamic modules",` 
match: `post processing of dynamic modules`
Which is not what is intended.

I suggest trying out the regexp on [regex101](https://regex101.com/r/DPUb8a/1).
<img width="693" alt="image" src="https://github.com/user-attachments/assets/b605b01f-0573-4753-83f6-39051a11748d" />

Reply:
@Jason3S Hello! I tightened up the regular expression to look for occurrences matching `"pre<event>":` and `"post<event>":`.

Reply:
Great! Thank you.